### ISSUE TITLE
Import functionality is almost broken

### ISSUE THREAD

**Issue by theAkito** [thunderbird/thunderbird-android#5106]
**Describe the bug**

**To Reproduce**
### 1
1. Export k9s_settings on older App version.
2. Remove older App version.
3. Install build v5.727.
4. Have firewall block App from internet.
5. Import all previously exported settings.
6. (Optional) Enable Internet access for App.
7. All Mail accounts have only an empty "Outbox" folder.

### 2
1. Export k9s_settings on older App version.
2. Remove older App version.
3. Install build v5.727.
4. Let App have internet access.
5. Import all previously exported settings.
6. Provide wrong password for any amount of mail accounts.
7. All Mail accounts, that were provided with a wrong password, have only an empty "Outbox" folder.
8. Remove faulty Mail account.
9. Re-import this faulty mail account, only.
10. The empty "Outbox" folder is now the default folder on top of all folders.
11. Every time you change to this mail account, the "Outbox" folder will _always_ be the first one opened on top.
12. There is no way to set the "Inbox" folder back to top.

This is only a summary of all the issues. There are more very similar issues, that I do not want to repeat here.
In short, the Import functionality is almost broken; if you just make 1 mistake, you have to uninstall and re-install the whole application. This is no way to treat imports.

**Expected behavior**
Importing works smoothly.

**Environment (please complete the following information):**
 - K-9 Mail version: v5.727
 - Android version: 11
 - Device: POCO X3
 - Account type: Generic IMAP

**Additional context**
The "stable" version always crashes, when I try to import settings. That's why I am using the newest "Pre-Release" version.

**Logs**
Please take some time to [retrieve logs](https://github.com/k9mail/k-9/wiki/LoggingErrors) and attach them here:
No, I will not uninstall and re-install the App for the 7th time.

**Comment by cketti** [comment]
Looks like we're not fetching the folder list from the server when there's only the local Outbox folder. You can work around this by going to `Account settings > Fetching mail > Incoming server` and pressing the "Next" button.
